docker-compose.yml

```diff
@@ -26,10 +26,10 @@ services:
       # You can grab these for dev with:
       #   cf t -s sandbox
       #   cf env funding-service-design-application-store-dev
```

Comment:
I think this should be `cf service-key form-uploads-dev local-testing` ?  you didn't do changes here but uncommented the AWS creds so want to be sure.